### PR TITLE
Improve wording on question

### PR DESCRIPTION
**Pull request by 3lvis**

### CODE REVIEW

File: GIT_AND_GITHUB.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ discuss other alternatives. Of course, this means you are putting yourself at ri
 of sharing your knowledge and/or learning some new stuff, so please be careful not
 to end up being even more awesome than you are.
 
-#### I need the code in my last pull request in my next. What do?
+#### My pull request depends on another pending pull request. What to do?
```
Comment (sindrenm) [inline GIT_AND_GITHUB.md]: Much better. I think I would also go with “what do I do” for the question.

Comment (jgorset) [inline GIT_AND_GITHUB.md]: Yeah, that'd make it _perfect_!

**Comment by 3lvis** [comment]
@sindrenm Should be good now 😄

**Comment by sindrenm** [comment]
You rock. :+1:

**Pull request merged**